examples/cpp_classification/classification.cpp

```diff
@@ -1,13 +1,19 @@
 #include <caffe/caffe.hpp>
+#ifdef USE_OPENCV
```

Comment:
Change the filename to image_classification since there will be text, audio and video classifications.